src/components/Header.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormEvent, useState } from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  addTodo: (title: string) => void,
+  isInputDisabled: boolean,
+};
+
+export const Header: React.FC<Props> = ({
+  todos, addTodo, isInputDisabled,
+}) => {
```

Comment:
Fix it everywhere please
```suggestion
export const Header: React.FC<Props> = ({
  todos, 
  addTodo, 
  isInputDisabled,
}) => {
```

Reply:
thank you, done